micro-ui/web/micro-ui-internals/packages/modules/workbench/src/components/DigitJSONForm.js

```
@@ -255,111 +233,221 @@
+            )}
+          </Form>
+        </Card>
+        {showToast && (
+          <Toast
+            label={t(showToast)}
+            error={showErrorToast}
+            onClose={() => {
+              setShowToast(null);
+            }}
+            isDleteBtn={true}
+          ></Toast>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)